config/global.js

```diff
@@ -42,7 +42,7 @@ const {
 	SC_SUPERHERO_USER_NAME = 'superhero@schul-cloud.org',
 	HELPAREA_URL,
 	FEATURE_MATRIX_MESSENGER_ENABLED,
-	FEATURE_MESSENGER_SCHOOL_SETTINGS_VISIBLE,
+	MATRIX_MESSENGER__SCHOOL_SETTINGS_VISIBLE,
```

Comment:
Why is this flag duplicated in global.js, it should be accessed using the configuration everywhere.